spec/features/creating_peep_spec.rb

```diff
@@ -0,0 +1,25 @@
+feature "creating a peep" do
+
+  let(:user) do
+    user = User.create(email: 'coco@gmail.com',
+                  password: 'lila',
+                  password_confirmation: 'lila',
+                  name: 'Julien',
+                  user_name: 'coco')
+  end
+
+  before do
+    Timecop.freeze(Time.local(2015))
+  end
+
+  after do
+    Timecop.return
+  end
+
+  scenario "as a signed in user" do
+    expect{create_peep(email: user.email,password: user.password,content: 'Nadal won the final')}.to change(Peep, :count).by(1)
```

Comment:
Metrics/LineLength: Line is too long. [127/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)